### PR TITLE
Mati branch

### PR DESCRIPTION
**Pull request by mggaray**
**Branch with SQL creation and Allergies, Dishes, Foods Controllers**

### CODE REVIEW

File: src/controllers/base-sql.js [hunk 1]
```diff
@@ -105,7 +105,7 @@ export default class BaseSQLController {
 			if (!error) {
 				onSuccess({ message: `Your ${this.#SingularEntityId} was deleted successfully` });
 			} else {
-				onError({ message: 'Sorry we have an unexpected error', error: err.sqlMessage });
+				onError({ message: 'Sorry we have an unexpected error', error: error.sqlMessage });
```
Comment (julian-er) [inline src/controllers/base-sql.js:108]: Thanks for fixing this!

File: src/controllers/dishes.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+	//#region PUT methods
+	/**
+	 * Edits a dish
+	 * @param {Request} req The Express request
+	 * @param {Response} res The Express response
+	 */
+	editDish(req, res) {
+		const { id } = req.params;
+		const query =
```
Comment (julian-er) [inline src/controllers/dishes.js]: Please, put it inline

File: src/controllers/dishes.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+	//#region POST methods
+	/**
+	 * Creates a dish
+	 * @param {Request} req The Express request
+	 * @param {Response} res The Express response
+	 */
+
+	createDish(req, res) {
+		const query =
```
Comment (julian-er) [inline src/controllers/dishes.js]: Please put it inline

File: src/controllers/foods.js [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+						})
+					);
+				}
+			});
+		});
+	}
+
+	//#endregion
+
+	/**
```
Comment (julian-er) [inline src/controllers/foods.js:55]: JSDoc to document your function should be done just before your function starts. Here is before a #region line and then you have your function

**Review by julian-er — COMMENTED** [review]
Please fix these minor things and then i accepts your PR

**Comment by mggaray** [comment]
Fixed comments

File: src/controllers/foods.js (hunk 4 above)
Comment (mggaray) [reply src/controllers/foods.js:55]: Fixed in the last commit

File: src/controllers/dishes.js (hunk 3 above)
Comment (mggaray) [reply src/controllers/dishes.js]: Fixed in the last commit

File: src/controllers/dishes.js (hunk 2 above)
Comment (mggaray) [reply src/controllers/dishes.js]: Fixed in the last commit

**Review by julian-er — APPROVED** [review]
Good job thanks

**Comment by AdanVolken** [comment]
Passed

**Comment by AdanVolken** [comment]
Nice, sorry for the closed it was a mistake!

**Review by AdanVolken — APPROVED** [review]

**Pull request merged**